aten/src/ATen/native/Resize.cpp

```diff
@@ -122,25 +125,30 @@ const Tensor& resize_as_(
   return result;
 }
 
+TensorImpl* resize_impl_cpu_(
+    TensorImpl* self,
+    IntArrayRef size,
+    c10::optional<IntArrayRef> stride,
+    bool resize_storage) {
+  return resize_impl_template_<
+      &maybe_resize_storage_cpu,
+      &compute_storage_size_default>(self, size, stride);
+}
+
 const Tensor& resize_(
     const Tensor& self,
     IntArrayRef size,
     c10::optional<MemoryFormat> optional_memory_format) {
-  if (self.has_names()) {
-    return resize_named_tensor_(self, size, optional_memory_format);
-  }
-  auto* self_ = self.unsafeGetTensorImpl();
-  // NOLINTNEXTLINE(bugprone-argument-comment)
-  resize_impl_cpu_(self_, size, /*strides=*/c10::nullopt);
-  if (optional_memory_format.has_value()) {
-    auto memory_format =
-        optional_memory_format.value();
-    TORCH_CHECK(
-        memory_format != MemoryFormat::Preserve,
-        "Unsupported memory format",
-        memory_format);
-    self_->empty_tensor_restride(memory_format);
-  }
+  resize_template<&resize_impl_cpu_>(self, size, c10::nullopt, optional_memory_format, true);
```

Comment:
nit: it will be nice to pass the keywords of these arguments like: `/*arg_name=*/ true`, for code readability.

```cpp
resize_template<&resize_impl_cpu_>(self, size, /*strides=*/ c10::nullopt, optional_memory_format, /*resize_storage=*/ true);
```